demos/handwritten_text_recognition_demo/python/data/test_img_id_gt.txt

```diff
@@ -0,0 +1,2189 @@
+eng_AF_004_line0.jpg,DFI Rocks
```

Comment:
we do not store annotations as part of demos

Reply:
Removed.